### PR TITLE
Update channel to configuration channel in channel.js

### PR DESCRIPTION
**Pull request by DeepaliAloori**
https://github.ibm.com/alchemy-containers/satellite-config/issues/1074

### CODE REVIEW

File: app/apollo/resolvers/channel.js [hunk 1]
```diff
@@ -300,10 +300,10 @@ const channelResolvers = {
       if(versionNameExists) {
         throw new RazeeValidationError(context.req.t('The version name {{name}} already exists', {'name':name}), context);
       }
-      // validate the number of total channel versions are under the limit
+      // validate the number of total configuration channel versions are under the limit
       const total = await models.DeployableVersion.count({org_id, channel_id: channel_uuid});
       if (total >= CHANNEL_VERSION_LIMITS.MAX_TOTAL ) {
-        throw new RazeeValidationError(context.req.t('Too many channel version are registered under {{channel_uuid}}.', {'channel_uuid':channel_uuid}), context);
+        throw new RazeeValidationError(context.req.t('Too many configuration channel version are registered under {{channel_uuid}}.', {'channel_uuid':channel_uuid}), context);
```
Comment (dalehille) [inline app/apollo/resolvers/channel.js:306]: `version` should change to `versions`

**Review by dalehille — CHANGES_REQUESTED** [review]

File: app/apollo/resolvers/channel.js (hunk 1 above)
Comment (DeepaliAloori) [reply app/apollo/resolvers/channel.js]: updated

File: app/apollo/resolvers/channel.js [hunk 2]
```diff
@@ -145,7 +145,7 @@ const channelResolvers = {
         const channelFilter = channelName ? { name: channelName, org_id } : { uuid: channelUuid, org_id } ;
         const channel = await models.Channel.findOne(channelFilter);
         if(!channel){
-          throw new NotFoundError(context.req.t('Could not find the channel with uuid/name {{channelUuid}}/channelName.', {'channelUuid':channelUuid}), context);
+          throw new NotFoundError(context.req.t('Could not find the configuration channel with uuid/name {{channel_uuid}}/channelName.', {'channel_uuid':channel_uuid}), context);
```
Comment (dalehille) [inline app/apollo/resolvers/channel.js:148]: this should be 
         ` throw new NotFoundError(context.req.t('Could not find the configuration channel with uuid/name {{channelUuid}}/channelName.', {'channelUuid':channelUuid}), context);`

Comment (DeepaliAloori) [reply app/apollo/resolvers/channel.js]: modified.

**Comment by dalehille** [comment]
This looks to be pulling in an older version of master into your branch.  Can you try these steps to see if your branch gets updated with the latest?
```
git checkout master
git pull origin master
git checkout DeepaliAloori-patch-1-1
git merge master
```

**Review by dalehille — APPROVED** [review]

**Pull request merged**